### DISCUSSION TITLE
Show Search Result on map missing

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#604] [Q&A] [answered]
I don't find any more the option "show search results on the map". Did this disappear?

**Comment by devemux86** [comment]
@DolomitesTom Do you search in favorites or search in POI or click a POI category?

"Show on map" appears as option at top of the results when search in POI or click a POI category.

**Reply by DolomitesTom** [reply]
Not in POi, also no standard category, since this takes too long, have searched for "Biergarten". Can this result on maps be shown for every search result, independent of the search settings?

**Reply by devemux86** [reply]
@DolomitesTom is it an online search?

Can you post a screenshot of the results to see their type?

---

> since this takes too long

How many maps do you have activated?

It is better to have only the needed map for planning or navigation (most of the time 1 map is enough).
So the search will be much faster.

**Comment by DolomitesTom** [comment]
I have activated Germany, France and Luxembourg, the countries where I mostly make daytrips. But I have deactivated POI search. But even when I activate POI search I don't have the show on maps function.

<img src="https://github.com/user-attachments/assets/568c84b4-004d-48bf-b2d8-9d905ccd15a4" width="50%"/><img src="https://github.com/user-attachments/assets/9f95aef7-5a7f-4636-93f0-b5698f85928a" width="50%"/>

**Reply by devemux86** [reply]
@DolomitesTom thanks for the details.

1. Image shows favorites (they are already visible).

2. Image shows favorites + online search results.

"Show on map" does not seem to appear when there is a mix of favorites and other results (offline or online).
We could probably display it also in this case for the offline or online results.

**Reply by DolomitesTom** [reply]
yes please. But why not additionally display "show on map" and highlight with a blue dot also in case 1? Because I don't want to lose my previous map zoom, but show all results within the map section. Otherwise I have to select an individual search result and when I do this, it zooms on this focus.

**Reply by devemux86** [reply]
Case 1 has favorites in results. Search is done in visible favorites, so the favorites are already visible on the map.
You just have to click the favorite in results that want the map to scroll onto it.

Alternatively a "Show on map" in this case would scroll / zoom the map to display the bounds of all favorites in the results.
Do you mean this?

> Because I don't want to lose my previous map zoom

Next Cruiser anyway will avoid the zoom in such cases, so only scroll map on the selected item will be performed.
So the current zoom will not change.

**Comment by umpfmpf** [comment]
I have the same, but with the zoom less than 11,5 it will work for me.

**Comment by DolomitesTom** [comment]
> Case 1 has favorites in results. Search is done in visible favorites, so the favorites are already visible on the map. You just have to click the favorite in results that want the map to scroll onto it.

ok

> Alternatively a "Show on map" in this case would scroll / zoom the map to display the bounds of all favorites in the results.
Do you mean this?

I don't think so. I mean to show the results wich fit into the actual map zoom without zooming, especially for online results. Maybe not easy to realize.


> Next Cruiser anyway will avoid the zoom in such cases, so only scroll map on the selected item will be performed.
So the current zoom will not change.

I hope only when the function "show on map" is available, especially for online results. When an individual search result is selected, a scrolling and zooming in to this location is ok.

**Reply by devemux86** [reply]
> I mean to show the results wich fit into the actual map zoom without zooming, especially for online results.

- Offline POI search is done in the visible map area for performance.

- Online search is done without bounds by external servers.
"Show on map" usually needs to zoom out to show all results.

Unless we want the online search to also be done inside the visible map area.

**Reply by DolomitesTom** [reply]
Yes, if it is possible to have online search also inside the visible map area.

**Comment by devemux86** [comment]
"Show on map" function will be available for all search results (offline or online) in next Cruiser.

**Comment by devemux86** [comment]
"Show on map" function is now available for all search results (offline or online):

- https://github.com/devemux86/cruiser/discussions/610

**Comment by devemux86** [comment, marked as answer]
- "Show on map" function is now available for all search results (offline or online) in https://github.com/devemux86/cruiser/discussions/641
